modules/Bio/EnsEMBL/Analysis/Hive/Config/HomologyRnaseq.pm

```diff
@@ -0,0 +1,322 @@
+
+    'rnaseq_refine_db_server'       => $self->o('databases_server'),
+    'rnaseq_refine_db_port'         => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'              => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+    'production_db_server'         => 'mysql-ens-meta-prod-1',
+    'production_db_port'           => '4483',
+
+#    databases_to_delete => ['genblast_db', 'rnaseq_refine_db',],
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+    'min_toplevel_slice_length'   => 250,
+    genome_dumps                  => catdir($self->o('output_path'), 'genome_dumps'),
+    faidx_genome_file             => catfile($self->o('genome_dumps'), $self->o('species_name').'_toplevel.fa'),
+
+########################
+# Extra db settings
+########################
+    'num_tokens' => 10,
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+########################
+# db info
+########################
+    'genblast_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_rnaseq_support_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_gb_rnaseq_'.$self->o('release_number'),
+      -host   => $self->o('genblast_rnaseq_support_db_server'),
+      -port   => $self->o('genblast_rnaseq_support_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_rnaseq_support_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_gb_rnaseq_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_rnaseq_support_db_server'),
+      -port   => $self->o('genblast_rnaseq_support_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'rnaseq_refine_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_refine_'.$self->o('release_number'),
+      -host   => $self->o('rnaseq_refine_db_server'),
+      -port   => $self->o('rnaseq_refine_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+  };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+  return [
+    # inheriting database and hive tables' creation
+    @{$self->SUPER::pipeline_create_commands},
+  ];
+}
```

Comment:
I would have removed it because it only calls the parent method. It could stay as it could make it easier if we need to add commands

Reply:
I'll leave it then